docs/source/tutorial/tutorial-subscriptions.mdx

```diff
@@ -146,7 +146,9 @@ Next, update the `init()` method to the following:
 
 ```swift title="LaunchListViewModel.swift"
 init() {
-    startSubscription()
+    Network.shared.apollo.fetch(query: LaunchListQuery(cursor: nil)) { result in
```

Comment:
Same as other comment, this query code shouldn't be here after the section for executing your first query.